src/js/components/types.js

```diff
@@ -89,4 +90,5 @@ export const TerminalDefaultProps = {
   promptSymbol: '>',
   plugins: [],
   shortcuts: {},
+  afterChangeHandler: null,
```

Comment:
not needed if you switch to the typeof check

Reply:
Sorry to ask, but won't this throw up because afterChange is not required, I need to specify a default type no?